docs/data/system/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -37,13 +37,27 @@ You can follow progress on https://github.com/mui/material-ui/issues/30671.
 
 ### Grid
 
-The `Grid` component was updated to match the API of the new `PigmentGrid` component, to allow interoperability between the two:
+The `Grid` component was updated and stabilized:
 
-- The previous size and offset props were replaced with the `size` and `offset` props
+- The previous size (`xs`, `sm`, `md`, ...) and offset (`xsOffset`, `smOffset`, `mdOffset`, ...) props, which were named after the theme's breakpoints, were replaced with the `size` and `offset` props.
 - The spacing mechanism was reworked to use the `gap` CSS property.
 
 This brings some breaking changes described in the following sections.
 
+#### Stabilized API
```

Comment:
```suggestion
```
I think this heading is not necessary and the snippet below should come before mentioning the props changed.

Reply:
I added sections because they're different breaking changes so providing structure makes it easier to follow, IMO. The other sections are for the size and offset props, and for the removal of `disableEqualOverflow`.